can_tools/bootstrap_data/covid_variables.csv

```diff
@@ -8,6 +8,7 @@ deaths,rolling_average_7_day,people
 hospital_beds_in_use_covid,cumulative,beds
 hospital_beds_in_use_covid,new,beds
 hospital_beds_in_use_covid,current,beds
+hospital_beds_in_use_covid,cumulative,people
```

Comment:
@cc7768 is this true?

Didn't we plan to write a function that would try to insert rows and apply an `ON CONFLICT TO NOTHING` clause at the end?

Reply:
I suspect you asking if it is true is probably a nice way of saying that it isn't true -- I assumed that even that insert method would get upset because of the ordering, but, if not, then let's definitely keep grouping similar variables

Reply:
Okay, I will leave this as-is, but let me know if it causes any issues after merging.